content/courses/Ada_For_The_C_Embedded_Developer/chapters/06_Translation.rst

```diff
@@ -417,8 +417,8 @@ names in their type.
 In Ada, the package is a namespace |mdash| two entities declared in two
 different packages are clearly identified and can always be specifically
 designated. The C names are usually a good indication of the names of the
-futures packages and should be stripped |mdash| it will be possible to force
-full name if useful. For example, here's how the following declaration and call
+futures packages and should be stripped |mdash| it is possible to the full name
```

Comment:
It should probably be:

"it is possible to **force** the full name" 

instead.

Reply:
Is there actually a mechanism to force such adoption? Or is it just that the usage of the full name is permitted? I definitely made a typo here though that needs to be fixed.